lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/filteredStringListView.py

```diff
@@ -60,6 +61,11 @@ def __init__(self, items: Sequence[str] = None, parent=None):
 
         self.selectionModel().selectionChanged.connect(lambda: self.selectedItemsChanged.emit())
 
+        self.placeholder_label = QLabel("Drag objects here or click “+” to add", self)
+        self.placeholder_label.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        self.placeholder_label.setStyleSheet("color: gray; font-size: 18px;")
```

Comment:
I'm a bit worried about this specific color and pixel size here to work on max as well with different dpi monitors